### PR TITLE
[PIL-690][AV] - Integration Points b/w Gov.uk and Pillar 2

### PR DESCRIPTION
**Pull request by aashik-varahan**

### CODE REVIEW

File: app/views/bta/NoPlrIdGuidanceView.scala.html [hunk 1]
```diff
@@ -29,7 +29,7 @@ <h1 class="govuk-heading-l">@messages("bta.noPlrIdGuidance.heading")</h1>
     <p class="govuk-body">@messages("bta.noPlrIdGuidance.p1")</p>
 
     <p class="govuk-body">
-        <a href="@appConfig.howToRegisterPlr2GuidanceUrl" class="govuk-link govuk-link--no-visited-state" target="_blank"> @messages("bta.noPlrIdGuidance.link")</a>
+        <a href="@appConfig.startPagePillar2Url" class="govuk-link govuk-link--no-visited-state" target="_blank"> @messages("bta.noPlrIdGuidance.link")</a>
```
Comment (jassalrichy) [inline app/views/bta/NoPlrIdGuidanceView.scala.html:32]: appConfig.howToRegisterPlr2GuidanceUrl

File: app/views/errors/AlreadyRegisteredView.scala.html [hunk 2]
```diff
@@ -15,18 +15,21 @@
...
-<p class="govuk-body">@messages("alreadyRegistered.message1")</p>
-<p class="govuk-body">
-    @Html(messages("alreadyRegistered.message2", s"<a class='govuk-link' href='mailto:${appConfig.pillar2mailbox}'>${appConfig.pillar2mailbox}</a>"))
-</p>
 
-}
\ No newline at end of file
+@heading(messages("alreadyRegistered.heading"), "govuk-heading-l")
+@paragraphBody(messages("alreadyRegistered.message1"))
+@paragraphBody(messages("alreadyRegistered.message2"))
```
Comment (jassalrichy) [inline app/views/errors/AlreadyRegisteredView.scala.html:33]: Should this be @paragraphBodyLink and add the missing link params.

File: app/views/bta/NoPlrIdGuidanceView.scala.html (hunk 1 above)
Comment (aashik-varahan) [reply app/views/bta/NoPlrIdGuidanceView.scala.html:32]: This is changed as per the ticket requirement - AC8

File: conf/messages.en [hunk 3]
```diff
@@ -758,22 +759,22 @@ unauthorised.org.standard.heading = Sorry, you’re unable to use this service
 unauthorised.org.standard.message = You’ve signed in with a standard organisation account.
 unauthorised.org.standard.message2 = Only Government Gateway accounts with an administrator role can register to use this service.
 unauthorised.org.standard.message3 = You need to find someone with an administrator’s Government Gateway user ID who can register and then give you authority to report and manage pillar 2 top-up taxes.
-unauthorised.org.standard.hyperlink = Read the draft guidance for more information on who can use this service.
+unauthorised.org.standard.hyperlink = Find out more about who can report for Pillar 2 top-up taxes
 
 unauthorised.agent.heading = Sorry, you’re unable to use this service
-unauthorised.agent.message = You’ve signed in using an agent’s Government Gateway user ID. Agents cannot use this service to register or administer a group’s Pillar 2 top-up taxes.
-unauthorised.agent.message2 = If you are an agent who has been given authorisation to report Pillar 2 top-up taxes, you must
+unauthorised.agent.message = You've signed in using an agent's Government Gateway user ID. Only groups can register to use this service.
```
Comment (jassalrichy) [inline conf/messages.en:765]: incorrect apostrophes

File: conf/messages.en [hunk 4]
```diff
@@ -59,10 +59,11 @@ viewAmendSubscriptionFailed.link= Return to your account homepage to try again
 ###############################################
 
 
+
 alreadyRegistered.title= Your group is already registered
 alreadyRegistered.heading= Your group is already registered
-alreadyRegistered.message1= You will need to contact your administrator to give you access.
-alreadyRegistered.message2= You can email {0} if you are having problems accessing the service.
+alreadyRegistered.message1= You can find your group’s Pillar 2 top-up taxes ID within your business tax account.
+alreadyRegistered.message2= If you need to request access to this account, your administrator can give you access using 'Manage account' settings within their business tax account.
```
Comment (jassalrichy) [inline conf/messages.en:66]: incorrect apostrophes

File: conf/messages.en (hunk 3 above)
Comment (aashik-varahan) [reply conf/messages.en]: Changed to curly apostrophes

File: conf/messages.en [hunk 5]
```diff
@@ -59,10 +59,11 @@ viewAmendSubscriptionFailed.link= Return to your account homepage to try again
 ###############################################
 
 
+
 alreadyRegistered.title= Your group is already registered
 alreadyRegistered.heading= Your group is already registered
-alreadyRegistered.message1= You will need to contact your administrator to give you access.
-alreadyRegistered.message2= You can email {0} if you are having problems accessing the service.
+alreadyRegistered.message1= You can find your group’s Pillar 2 top-up taxes ID within your business tax account.
+alreadyRegistered.message2= If you need to request access to this account, your administrator can give you access using ’Manage account’ settings within their business tax account.
```
Comment (jassalrichy) [inline conf/messages.en:67]: use correct opening quote for ‘Manage account’

**Review by jassalrichy — DISMISSED** [review]

**Pull request merged**